Client/mods/deathmatch/logic/CClientRenderElementManager.cpp

```diff
@@ -11,6 +11,12 @@
 #include "StdInc.h"
 #include "CClientVectorGraphic.h"
 
+namespace
+{
+    const std::int64_t TEMPORARY_TEXTURES_CLEANUP_THRESHOLD = 10000ll;
+    const std::size_t TEMPORARY_TEXTURES_DELETE_THRESHOLD = 10u;
+}
+
```

Comment:
Has `namespace` any effect here?
+ `constexpr`

Reply:
The unnamed namespace in this context is the equivalent to `static` specifier to keep these variables inside a translation unit. Just the modern way of doing it. https://en.cppreference.com/w/cpp/language/namespace#Unnamed_namespaces

Reply:
Could you explain what it means practically? Afaik these variables will be inside this cpp without a namespace too.

Reply:
Example:

one.cpp
`int varA = 10;`

two.cpp
`int varA = 10;`

This will cause a compile time error: `one or more multiply defined symbols found.`
And to prevent it you have to use `static` or an unnamed namespace.

Reply:
You are right. I didn't have issues with multiply defined symbols before.
I checked it. `constrexpr` also prevents this error without a namespace.

Reply:
Yeah, okay.